app/frontend/src/components/forms/TeamManagement.vue

```diff
@@ -1,16 +1,25 @@
 <template>
   <div>
-    <!-- search input -->
-    <div class="team-search mt-6 mt-sm-0">
-      <v-text-field
-        v-model="search"
-        append-icon="mdi-magnify"
-        label="Search"
-        single-line
-        hide-details
-        class="pb-5"
-      />
-    </div>
+    <v-row class="mt-6">
+      <v-col cols="12" sm="8">
+        <h1 class="inline">Team Management</h1>
+      </v-col>
```

Comment:
I suggest changing `sm="4"` and leveraging a `<v-spacer />` element instead of just stretching the Team Management title object.
https://vuetifyjs.com/en/api/v-spacer/

Reply:
i like this suggestion.. but it didnt seem to display as well.